### ISSUE TITLE
Warnings

### ISSUE THREAD

**Issue by simlu** [valeriansaliou/node-fast-ratelimit#4]
Getting the following warnings:

```
...\node_modules\hashtable-patch-valeriansaliou\src\hashtable.h(6): warning C4067: unexpected tokens following preprocessor directive - expected a newline (compiling source file ..\src\hashtable.cpp) [...\node_modules\hashtable-patch-valeriansaliou\build\native.vcxproj]

...\node_modules\hashtable-patch-valeriansaliou\src\v8_value_hasher.h(7): warning C4067: unexpected tokens following preprocessor directive - expected a newline (compiling source file ..\src\hashtable.cpp) [...\node_modules\hashtable-patch-valeriansaliou\build\native.vcxproj]

..\src\hashtable.cpp(268): warning C4244: 'argument': conversion from 'double' to 'float', possible loss of data [...\node_modules\hashtable-patch-valeriansaliou\build\native.vcxproj]
```

**Comment by valeriansaliou** [comment]
Can you try pulling https://github.com/valeriansaliou/node-hashtable and editing the file to try to fix the issue (compile this one directly)?

I don't have the warnings here.

**Comment by valeriansaliou** [comment]
Hello there! I've fixed that in `v3.0.0`, using the JS-native Map object, which has as well be measured in my tests to be faster anyway. This new version does not compile any dependencies anymore, fixing this issue.
